dvr_scan/scanner.py

```diff
@@ -293,11 +305,11 @@ def scan_motion(self):
             if self.frames_total < 0:
                 self.frames_total = 0
             progress_bar = tqdm.tqdm(
-                total = self.frames_total, unit = ' frames',
-                desc = "[DVR-Scan] Processed")
+                total=self.frames_total, unit=' frames',
+                desc="[DVR-Scan] Processed")
 
         # Motion event scanning/detection loop.
-        while True:
+        while self.running:
```

Comment:
This is the only change i made to the scanner.py's functionality. I had to do it because in the GUI the scan run's in a seperate thread. Else the GUI freezes and for some reason when you kill the thread the scan keeps running.